src/Engine/StreamStorm/ai/PydanticAI.py

```
@@ -0,0 +1,95 @@
+            "",
+        )
+
+        agent = Agent(
+            model=self.model,
+            output_type=AIGeneratedChannels,
+            system_prompt=system_prompt,
+        )
+
+        try:
+            response = await self._generate(agent, prompt)
+        except Exceptionas as e:
```

Ruff (0.14.10)

Undefined name `Exceptionas`

(F821)